docs/guides/docs/testnet-migration/beta-3-to-beta-4-migration.mdx

```diff
@@ -0,0 +1,227 @@
+---
+title: Beta 4 Testnet Migration 
+category: Testnet Migration
+parent:
+  label: Guides
+  link: /guides
+---
+
+# Beta 4 Migration
+
+This guide focuses on the transition from the end of tool support for `beta 3` to the start of `beta 4`.
+
+- [Fuelup v0.19.4](https://github.com/FuelLabs/fuelup/releases/tag/v0.19.4)
+- [Sway v0.44.0](https://github.com/FuelLabs/sway/releases/tag/v0.44.0)
+- [TS SDK v0.51.0](https://github.com/FuelLabs/fuels-ts/releases/tag/v0.51.0)
+- [Rust SDK v0.47.0](https://github.com/FuelLabs/fuels-rs/releases/tag/v0.46.0)
```

Comment:
The link is pointing to `v0.46.0` but the bullet point mentions `v0.47.0` which is indeed the latest version.